activity_tracker/__init__.py

```diff
@@ -1,2 +1,2 @@
 __version__ = '0.0.1'
-from .tracker import ActivityTracker
+from .tracker import ActivityTracker  # noqa
```

Comment:
Instead of `# noqa`, could you just add `__all__ = ['ActivityTracker']`? I think it makes it a tiny bit more obvious it's imported for use by other modules, and flake8 is happy it.